content/robot_players.tex

```diff
@@ -17,7 +17,7 @@ \subsection{Hardware}
     \item A memory stick may remain in the head during operation.  Only ordinary USB flash memory keys that sit flush or recessed to the head casing may be utilized. Other USB dongles or devices, as well as memory sticks that are not flush or recessed, are not permitted.
 \end{itemize}
 
-A computer with two monitors (one for GC and one for TCM) will be provided by the event organizers for the purpose of sending GameController messages to the robots and observing if no robot violates the rules for Wifi usage.
+A computer with two monitors (one for GC and one for TCM) will be provided by the event organizers for the purpose of sending GameController messages to the robots and observing if no robot violates the rules for wireless network usage.
```

Comment:
Should be not in this PR? Instead a commit on the rules themselves?

Reply:
Yeah, should have been, but I thought this PR has to be merged anyhow so I let it in.